tests/kernel/install_ltp.pm

```diff
@@ -165,14 +165,15 @@ curl --form upload=\@/tmp/ltp-runtest-files-$tag --form target=assets_public $ai
 curl --form upload=\@/root/openposix-test-list-$tag --form target=assets_public $aiurl/upload_asset/openposix-test-list-$tag
 %;
 
-    script_output($up_script);
+    script_output($up_script, 300);
 }
 
 sub install_from_git {
     my ($tag) = @_;
     my $url = get_var('LTP_GIT_URL') || 'https://github.com/linux-test-project/ltp';
     my $rel = get_var('LTP_RELEASE') || '';
-    my $configure = './configure --with-open-posix-testsuite --with-realtime-testsuite';
+    my $timeout     = check_var('ARCH', 's390x') || check_var('ARCH', 'aarch64') ? 7200 : 1440;
```

Comment:
You should make sure that MAX_JOB_TIME is increased over the default 7200 in the according test suites to prevent incompletes

Reply:
I know about `MAX_JOB_TIME`, I didn't bother to set test to something lower than this. If it's needed, the job must be cloned with changed `MAX_JOB_TIME`. In previous implementation there were variable for this timeout, so the value was 1 hour. Simplify it without variable I just use 2 hours, although that's more than `MAX_JOB_TIME`. Even on arm something more than 1 hour means something broken with SUT (maybe btrfs broken?), so more than 2 hours is a waste of time.